### PR TITLE
Ported composite addon tests

### PR DESCRIPTION
**Pull request by mridgway**

### CODE REVIEW

File: tests/base/mojito-test.js [hunk 1]
```diff
@@ -28,6 +28,20 @@ YUI.add('mojito-meta-addon', function(Y, NAME) {
 
 /*
  */
+YUI.add('mojito-composite-addon', function(Y, NAME) {
+    Y.namespace('mojito.addons.ac');
+    return function() {};
```
Comment (drewfish) [inline tests/base/mojito-test.js:33]: I've never seen a YUI.add() that had a return statement before.  What does that do?

Comment (mridgway) [inline tests/base/mojito-test.js:33]: Not sure, it was copy/pasted from @mojit0's examples.  I don't think it does anything to be honest.

**Comment by FabianFrank** [comment]
How can those tests be run? Using mojito test?

**Comment by mridgway** [comment]
Eventually yes, but not right now. We're gearing up for the new testing framework.

**Comment by mojit0** [comment]
New tests, and existing tests, are being moved to Arrow in anticipation of open sourcing that framework. At present all that work is being done in a separate branch with anticipation of merging it into develop when the tests are ported, Arrow is open sourced, and we've completed work to augment mojito test with a way to run both the tests existing customers have written and the new framework tests being written for Arrow.

**Comment by mojit0** [comment]
New tests, and existing tests, are being moved to Arrow in anticipation of open sourcing that framework. At present all that work is being done in a separate branch with anticipation of merging it into develop when the tests are ported, Arrow is open sourced, and we've completed work to augment mojito test with a way to run both the tests existing customers have written and the new framework tests being written for Arrow.

ss

**Pull request merged**